### PR TITLE
docs: add Getting Started menuitem

### PR DESCRIPTION
**Pull request by pimlie**
docs: add section about Framework documentation

As requested on Discord, split the menu in GettingStarted and Usage and add information about Frameworks where you can skip the Install

### CODE REVIEW

File: docs/guide/README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+For the latest version in the v1.x branch you can use:<br/>
+[https://unpkg.com/vue-meta@1/lib/vue-meta.js](https://unpkg.com/vue-meta@1/lib/vue-meta.js)
+
+Or you can replace `1` with the full version number you wish to use.
+
+If you include vue-meta after Vue it will install automatically
 
-Vue.use(Router)
-Vue.use(Meta)
+**Uncompressed:**
```
Comment (manniL) [inline docs/guide/README.md]: Technically not 100% correct :relaxed: 	

```suggestion
**Unminified (suggested only for dev):**
```

File: docs/guide/README.md [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
+## Package manager
+**Yarn**
+```sh
+$ yarn add vue-meta
 ```
 
-## Options
+**npm**
+```sh
+$ npm install vue-meta --save
```
Comment (manniL) [inline docs/guide/README.md]:
```suggestion
$ npm i vue-meta
```

Save is not necessary anymore.

File: docs/guide/README.md [hunk 3]
```diff
@@ -1,36 +1,48 @@
...
+```
 
-`vue-meta` allows a few custom options:
+### Install
+You need to explicitly install vue-meta when using a package manager
```
Comment (manniL) [inline docs/guide/README.md]: Suggestion: Add a sentence regarding frameworks again.

Comment (pimlie) [reply docs/guide/README.md]: You offered a finger but I took the whole hand. Its now a full warning block :)

File: docs/guide/README.md [hunk 4]
```diff
@@ -1,36 +1,54 @@
...
+```sh
+$ npm i vue-meta
 ```
 
-See the [API](/api/#plugin-options) for a description of the available plugin options
+### Install
+
+:::warning Using a framework?
+
+If you use a framework like Nuxt.js or Gridsome this step is most likely **not** required. Consult the [documentation](/guide/frameworks.html) of your framework for more information
```
Comment (armandfarr) [inline docs/guide/README.md]: If you use a framework like Nuxt.js or Gridsome, vue-meta comes pre-installed and this step is most likely **not** required.

File: docs/guide/frameworks.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+If you are using a framework which uses `vue-meta`, please make sure to consult their documentation first
+
+## Gridsome
+
+Proceed to the [Gridsome documentation](https://gridsome.org/docs)
+
+## Nuxt.js
+
+Proceed to the [Nuxt.js documentation](https://nuxtjs.org/api)
```
[the remark this answers is not in the document]
Comment (pimlie) [reply docs/guide/frameworks.md]: Thanks but I deliberately didnt add links to the specific doc pages of the frameworks as I prefer not having to update these links once nuxt or gridsome change them

**Pull request merged**